nsv13/code/modules/jobs/security/weapons.dm

```diff
@@ -107,6 +125,60 @@
 	build_path = /obj/item/ammo_casing/c9mm/rubber
 	category = list("initial", "Security")
 
+/datum/design/rubber9mm
+	name = "9mm ammobox (rubber)"
+	id = "glock_rubberammo"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 25000)
+	build_path = /obj/item/ammo_box/c9mm/rubber
+	category = list("Ammo")
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/glockmag
+	name = "Glock Magazine (rubber)"
```

Comment:
```suggestion
	name = "Glock Magazine (Rubber)"
```
The extra properties are also capitalized, same for the rest of the file

Reply:
@BlueHNT I still see other designs with uncapitalized names

Reply:
> @BlueHNT I still see other designs with uncapitalized names

Naming scheme

magazines
design Glock Magazine (Type)
obj pistol magazine (Type)

ammo boxes
design 9mm Ammo Box (Type)
obj ammo box (Type)